addons/base_vat/models/res_partner.py

```diff
@@ -167,6 +168,14 @@ def _construct_constraint_msg(self, country_code):
     __check_vat_ch_re1 = re.compile(r'(MWST|TVA|IVA)[0-9]{6}$')
     __check_vat_ch_re2 = re.compile(r'E([0-9]{9}|-[0-9]{3}\.[0-9]{3}\.[0-9]{3})(MWST|TVA|IVA)$')
 
+    def check_vat_au(self, vat):
+        """Check Australian Business Number."""
+        try:
+            from stdnum.au.abn import is_valid
```

Comment:
could do this in the import statements above (in order to avoid to do this again and again